docs/api/README.md

```diff
@@ -263,6 +265,57 @@ const takeLatest = (patternOrChannel, saga, ...args) => fork(function*() {
 
 You can also pass in a channel as argument and the behaviour is the same as [takeLatest(pattern, saga, ...args)](#takelatestpattern-saga-args).
 
+### `takeLeading(pattern, saga, ...args)`
+
+Spawns a `saga` on each action dispatched to the Store that matches `pattern`. And automatically ignores
+any coming new `saga` task if it's still running.
+
+Each time an action is dispatched to the store. And if this action matches `pattern`, `takeLeading`
```

Comment:
This paragraph described the behaviour again in slightly inaccurate way. Would be easier to say that it's listening for the actions when it doesn't run a saga.